harmonica/_equivalent_sources/cartesian.py

```diff
@@ -238,10 +263,14 @@ def _build_points(self, coordinates):
         """
         if self.block_size is not None:
             coordinates = self._block_average_coordinates(coordinates)
+        if self.depth == "default":
+            self.depth_ = 4.5 * np.median(vd.median_distance(coordinates, k_nearest=1))
```

Comment:
As discussed in #424, this should be the mean of the median distance
```suggestion
            self.depth_ = 4.5 * np.mean(vd.median_distance(coordinates, k_nearest=1))
```